stubs/gdb/gdb/events.pyi

```diff
@@ -0,0 +1,131 @@
+from collections.abc import Callable, Sequence
+
+import gdb
+
+class ThreadEvent:
+    inferior_thread: gdb.InferiorThread
+
+class ContinueEvent(ThreadEvent): ...
+
+class ContinueEventRegistry:
+    def connect(self, __object: Callable[[ContinueEvent], object]) -> None: ...
+    def disconnect(self, __object: Callable[[ContinueEvent], object]) -> None: ...
+
+cont: ContinueEventRegistry
+
+class ExitedEvent:
+    exit_code: int
+    inferior: gdb.Inferior
+
+class ExitedEventRegistry:
+    def connect(self, __object: Callable[[ExitedEvent], object]) -> None: ...
+    def disconnect(self, __object: Callable[[ExitedEvent], object]) -> None: ...
+
+exited: ExitedEventRegistry
+
+class StopEvent(ThreadEvent):
+    stop_signal: str
+
+class BreakpointEvent(StopEvent):
+    breakpoints = Sequence[gdb.Breakpoint]
```

Comment:
This line was just flagged by a new check I'm working on for [flake8-pyi](https://github.com/PyCQA/flake8-pyi), the linter we use at typeshed. Did you mean for this to be a type alias, or did you mean to have `breakpoints: Sequence[gdb.Breakpoint]`?

Reply:
It should be an attribute according to https://sourceware.org/gdb/onlinedocs/gdb/Events-In-Python.html (search for "BreakpointEvent.breakpoints"). For some reason I can't find it in the source code though.

Reply:
Thank you for spotting this! I confirm it should be an attribute. Re-reading this some time after writing it, I now also see a typo on line 31. I’ve just opened a PR to fix this: #8144